public/css/scss/modules/all.scss

```diff
@@ -0,0 +1,17 @@
+body {
+    font-family: 'Inter', sans-serif;
+    margin: 0;
+    padding: 0;
+    background: var(--grey-200);
+}
+
+* {
+    box-sizing: border-box;
+}
+
+@import "../pages/index.scss";
```

Comment:
Just a personal preference, but I would have a layouts/base or in your situation a pages/base where I would include everything from pages, rather then having them hidden in the modules folder. This way you wouldn't have to do something like '../pages/index'.

`
@import 'characters';
@import 'comics';
@import 'browse';
@import 'search';
@import 'offline';
`

Reply:
does this work if you don't name you file with an underscore?

Reply:
I do think so, yes.

Reply:
it didn't, renamed files with underscore

Reply:
ah my bad, I thought it was used for things like partials.